code42_connector.py

```diff
@@ -236,5 +314,16 @@ def main():
     exit(0)
 
 
+def _generate_password():
```

Comment:
Something about this being visible on an open source repo doesn't sit right with me. Maybe we should force users to supply a password, I'm going to talk to Tony.

Reply:
Why not just make password optional?  If not provided, leave the user in invited mode.

Reply:
Mainly because invited mode can be pretty confusing unless you actually go through the invite process. The user account is created, but doesn't show up in the normal org user list unless you click on the 'invited' tab. But I suppose that's just a problem in the product that we don't necessarily have to try to fix ourselves on this end.

Reply:
it's worth noting that the invite behavior is what we do in XSOAR currently (I believe). I did reach out to Tony about this and it's all fine by him but he recommended we reach out to a PAL. Personally I think using the invite pattern is probably fine if properly documented.

Reply:
We could add a note to the `verbose` saying that by not including a password, the user will be in `invite` mode.

Reply:
Sounds good, I doubt phantom will actually be used much for creating users either way.